packages/web/app/components/ImagingRequestsTable.jsx

```diff
@@ -143,7 +143,13 @@ export const ImagingRequestsTable = React.memo(({ encounterId, memoryKey, status
       autoRefresh={!encounterId}
       endpoint={encounterId ? `encounter/${encounterId}/imagingRequests` : 'imagingRequest'}
       columns={encounterId ? encounterColumns : globalColumns}
-      noDataMessage="No imaging requests found"
+      noDataMessage={
+        <TranslatedText
+          stringId="imaging.list.noData"
```

Comment:
```suggestion
          stringId="imaging.table.noData"
```
Id probably do this?